src/data/javascript-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const javascriptQuiz = [
+  {
+    Question: "In Javascript,let x='Prakhar' then what will be the value of console.log(x.substr(2,5),x.substring(2,5))?",
```

Comment:
Shorter questions work better when rendered on the quiz page. 

```suggestion
    Question: "In JavaScript, if let x='Prakhar', then what will be the result for console.log(x.substring(2,5))?",
```